README.md

```diff
@@ -18,9 +18,18 @@ for our rules and recommendations on contributions.
 
 ## Development
 
-### Dev Container
+#### 🌏  Open in the Cloud 
 
+Click any of the buttons below to start a new development environment to demo or contribute to the codebase without having to install anything on your machine:
+
+[![Open in VS Code](https://img.shields.io/badge/Open%20in-VS%20Code-blue?logo=visualstudiocode)](https://vscode.dev/github/yourls/website)
+[![Open in Glitch](https://img.shields.io/badge/Open%20in-Glitch-blue?logo=glitch)](https://glitch.com/edit/#!/import/github/yourls/website)
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=466833669)
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/yourls/website)
+[![Edit in Codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/yourls/website)
+[![Open in Repl.it](https://replit.com/badge/github/withastro/astro)](https://replit.com/github/yourls/website)
+[![Open in Codeanywhere](https://codeanywhere.com/img/open-in-codeanywhere-btn.svg)](https://app.codeanywhere.com/#https://github.com/yourls/website)
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/yourls/website)
```

Comment:
Gitpod "Classic" is headed to the cloud graveyard in just a couple months (April 2025). Gitpod "Flex" is ironically much _less_ flexible and doesn't offer a one-click managed setup; it has to run on (last I checked) a local Mac or your own AWS tenancy.

(Salt level: High. I was happily using "Classic" until the sunset announcement last year. The "Flex" shift turned me off of Gitpod entirely.)